f/fcs/fn_fcsInit.sqf

```diff
@@ -24,83 +24,93 @@
 
 */
 
-// Doesn't need to run on DS
-if (isDedicated) exitWith {};
+// Doesn't need to run on DS or HC
+if !(hasInterface) exitWith {};
 
-params ["_vehicle"];
+params ["_vehicle",["_mode",true]];
 
 if !(_vehicle isKindOf "LandVehicle") exitWith { diag_log "FCS: tried to run on something that isn't a vehicle" };
 	
-if (_vehicle getVariable ["fcs_hasEH",false]) exitWith { diag_log "FCS: tried to run on something that already has FCS set up"};
+if (_vehicle getVariable ["f3_fcs_hasEH",false]) exitWith { diag_log "FCS: tried to run on something that already has FCS set up"};
 
-// Commander's override action
-_vehicle addAction
-[
-	"Commander's override",	
-	{
-		params ["_target", "_caller", "_actionId", "_arguments"];
-		[_target,_caller] call f_fnc_fcsCommanderOverride;
-	},
-	nil,	
-	1.5,	
-	false,	
-	true,	
-	"",	
-	"(_this == commander _target) && {!(isNull gunner _target) && !(_target getVariable [""fcsCommanderOverride_cooldown"",false]) && !(_target getVariable [""fcs_failure"",false])}", 
-	0,		
-	false,	
-	"",	
-	""	
-];
+// If the FCS briefing tab hasn't been added already, add it.
+if (isNil "f3_fcs_briefingDone") then {
```

Comment:
Minor: my personal preference would be for this to be an `exitWith` at the top of `f/fcs/fn_fcsBriefing.sqf`, instead of it being the responsibility of the caller to check.

Reply:
I wanted to do it this way because fcsInit can be run _a lot_ depending on the vehicles in the mission, so saving the function call is [probably] better for performance.